playground/proto/playground.proto

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option go_package= "backend/pkg/api";
+package proto;
+
+enum sdk {
+  JAVA = 0;
+  GO = 1;
+  PYTHON = 2;
+  SCIO = 3;
+}
+
+// CodeInfo represents a code text and options of SDK which executes the code.
```

Comment:
Could you please change `CodeInfo` to `CodeObject` to match the message name below?

Reply:
resolved in proto PR